tests/delete/tests.py

```diff
@@ -356,6 +357,62 @@ def test_delete_with_keeping_parents(self):
         self.assertFalse(RChild.objects.filter(id=child.id).exists())
         self.assertTrue(R.objects.filter(id=parent_id).exists())
 
+    def test_qs_delete_returns_num_rows(self):
+        # test QuerySet .delete() method returns number of deleted rows
+        Avatar.objects.bulk_create([Avatar(desc='a'), Avatar(desc='b'), Avatar(desc='a')])
+        avatars_count = Avatar.objects.count()
+        deleted, rows_count = Avatar.objects.all().delete()
+        self.assertTrue(deleted)
+        self.assertEqual(deleted, avatars_count)
+
+        # test QuerySet .delete() method returns rows_count
+
+        r = R.objects.create()
+        h1 = HiddenUser.objects.create(r=r)
+        HiddenUser.objects.create(r=r)
+        HiddenUserProfile.objects.create(user=h1)
+        existed_objs = {
+            R._meta.label: R.objects.count(),
+            HiddenUser._meta.label: HiddenUser.objects.count(),
+            A._meta.label: A.objects.count(),
+            MR._meta.label: MR.objects.count(),
+            HiddenUserProfile._meta.label: HiddenUserProfile.objects.count()
```

Comment:
please include trailing comma in dictionary so if more items are added, we don't have to modify this line again